dev/Pager/InteractionTests/PagerTests.cs

```diff
@@ -0,0 +1,960 @@
+
+        [ClassInitialize]
+        [TestProperty("RunAs", "User")]
+        [TestProperty("Classification", "Integration")]
+        [TestProperty("Platform", "Any")]
+        [TestProperty("MUXControlsTestSuite", "SuiteB")]
+        public static void ClassInitialize(TestContext testContext)
+        {
+            TestEnvironment.Initialize(testContext);
+        }
+
+        public void TestCleanup()
+        {
+            TestCleanupHelper.Cleanup();
+        }
+
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void NumberBoxDisplayChangingPageTest()
+        {
+            using (var setup = new TestSetupHelper("Pager Tests"))
+            {
+                elements = new PagerTestsPageElements();
+
+                SetNumberBoxDisplayMode();
+                VerifyNumberBoxDisplayMode();
+                VerifyPageChangedEventOutput(0);
+
+                SendValueToNumberBox("3"); // Note: Pager displays numbers starting at 1 but the page changed event sends 0-based numbers
+                VerifyPageChangedEventOutput(2);
+
+                SendValueToNumberBox("1");
+                VerifyPageChangedEventOutput(0);
+
+                SendValueToNumberBox("5");
+                VerifyPageChangedEventOutput(4);
+
+                SendValueToNumberBox("2");
+                VerifyPageChangedEventOutput(1);
+
+                SendValueToNumberBox("100");
+                Verify.AreEqual("5", FindTextBox(elements.GetPagerNumberBox()).GetText()); // If over max, value should be clamped down to the max.
+                VerifyPageChangedEventOutput(4);
+
+                SendValueToNumberBox("-100");
+                Verify.AreEqual("1", FindTextBox(elements.GetPagerNumberBox()).GetText()); // If under min, value should be clamped up to the min.
+                VerifyPageChangedEventOutput(0);
+
+
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void ComboBoxDisplayChangingPageTest()
+        {
+            using (var setup = new TestSetupHelper("Pager Tests"))
+            {
+                elements = new PagerTestsPageElements();
+                VerifyPageChangedEventOutput(0);
+
+                SetComboBoxDisplayMode();
+                VerifyComboBoxDisplayMode();
+
+                SelectValueInPagerCombBox(2);
+                VerifyPageChangedEventOutput(2);
+
+                SelectValueInPagerCombBox(4);
+                VerifyPageChangedEventOutput(4);
+
+                SelectValueInPagerCombBox(0);
+                VerifyPageChangedEventOutput(0);
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void AutoDisplayChangingPageTest()
+        {
+            using (var setup = new TestSetupHelper("Pager Tests"))
+            {
+                elements = new PagerTestsPageElements();
+                VerifyPageChangedEventOutput(0);
+
+                VerifyAutoDisplayMode();
```

Comment:
>VerifyAutoDisplayMode [](start = 16, length = 21)

I would probably call SetAutoDisplayMode even if it is not needed

Reply:
Okay!